examples/wearable/UIComponents/contents/presentation-views/list/group-index/multi-left.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		Wearable UI
+	</title>
+	<link href="../../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
+	<link href="../../../../css/style.css" rel="stylesheet" />
+	<link href="../../../../css/style.circle.css" rel="stylesheet" />
+	<script src="../../../../js/circle-helper.js">
+	</script>
+	<script src="../../../../lib/tau/wearable/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="page">
+		<header class="ui-header">
+			<h2 class="ui-title">
+				Title
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li>
+					<a>
+						List 12
+					</a>
+				</li>
+				<li class="ui-li-group-index-left">
```

Comment:
I think you can try to do this in another way.
`<div class="ui-li-group-index ui-li-group-index-left">`

Reply:
Right. Changed examples and simplified styles. Thanks.